packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.10+1
```

Comment:
> > Our repo stops using prereleases after 1.0.0.
> 
> Is this true for all packages in this repo, or just the `google_maps_flutter` package ?

All packages. We follow [the official Dart recommendations](https://dart.dev/tools/pub/versioning#semantic-versions) in this regard.

> Because i saw pre-releases in the `image_picker` packages.

`image_picker` hasn't reached 1.0.